core/vm/memory.go

```diff
@@ -43,6 +48,20 @@ func (m *Memory) Set(offset, size uint64, value []byte) {
 	}
 }
 
+// Set32 sets the 32 bytes starting at offset to the value of val, left-padded with zeroes to
+// 32 bytes.
+func (m *Memory) Set32(offset uint64, val *big.Int) {
+	// length of store may never be less than offset + size.
+	// The store should be resized PRIOR to setting the memory
+	if 32 > uint64(len(m.store)) {
+		panic("INVALID memory: store empty")
```

Comment:
I think the fact you're panicing gets the message across already, no need for the caps :)

Reply:
Same as above